### ISSUE TITLE
marginPagesDisplayed uneven on left and right side

### ISSUE THREAD

**Issue by sainttx** [AdeleD/react-paginate#222]
When setting the marginPagesDisplayed value, the right side always shows 1 value more than the left.

Code used:
```  
<ReactPaginate
  containerClassName="pagination"
  pageClassName="page-item"
  pageLinkClassName="page-link"
  activeClassName="active"
  disabledClassName="disabled"
  previousClassName="page-item"
  previousLinkClassName="page-link"
  nextClassName="page-item"
  nextLinkClassName="page-link"
  breakClassName="page-item disabled"
  breakLabel={<a className="page-link">...</a>}
  pageCount={this.state.data.length / this.state.pageItems}
  pageRangeDisplayed={3}
  marginPagesDisplayed={1}
  initialPage={0}
  onPageChange={data => this.setState({ page: data.selected })}
/>
```
Picture:
![capture](https://user-images.githubusercontent.com/9153518/42477908-e52dd176-83a0-11e8-8feb-0405f7f74180.PNG)

The right side shows page 345, 346 while the left only shows page 1. With the code I used page 345 should not be displayed.

**Comment by AdeleD** [comment]
This bug should be fixed in the newer versions. Which version of react-paginate do you use?

**Comment by chuneekho** [comment]
i have the same issue in 5.2.4

**Comment by jinixx** [comment]
still not fixed in 6.3.0

**Comment by MonsieurV** [comment]
Seems to have been fixed (certainly from #302)

![image](https://user-images.githubusercontent.com/1753348/140183203-b4f4888e-0af5-4128-9a9f-5da08fac0f39.png)

with (on the demo)

```js
        <ReactPaginate
          ...
          pageCount={356}
          marginPagesDisplayed={1}
          pageRangeDisplayed={3}
        />
```

Before the `v7.2.0` is released, you can test and  install from GitHub:

```sh
npm install --save github:AdeleD/react-paginate#master
```

Please re-open with a code sample / MWE if you still have issues

Thanks for the report @sainttx & all
